src/chromatix/functional/lenses.py

```diff
@@ -65,6 +73,57 @@ def ff_lens(
     return optical_fft(field, f, n)
 
 
+def high_na_lens(
```

Comment:
Could we rename this to `high_na_ff_lens`?

Reply:
Agreed, done.